src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import '../styles/card';
+@import '../styles/stars';
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property should be applied globally to every element, not just the `body`. This helps to ensure that the width and height properties (and min/max properties) includes content, padding, and border, but not the margin. Consider moving this rule to a universal selector `*`.